modules/apps/callbacks/ibc_middleware_test.go

```diff
@@ -980,7 +980,7 @@ func (s *CallbacksTestSuite) TestOnChanCloseInit() {
 	icaControllerStack, ok := s.chainA.App.GetIBCKeeper().Router.GetRoute(icacontrollertypes.SubModuleName)
 	s.Require().True(ok)
 
-	controllerStack := icaControllerStack.(porttypes.Middleware)
+	controllerStack := icaControllerStack.(porttypes.ICS4Wrapper)
```

Comment:
This works fine on `SendPacket` or `WriteAcknowledgement` tests but it wont work on tests such `TestOnChanCloseInit` because ICS4Wrapper doesn't have `OnChanCloseInit` which is what the linter is saying